src/test/resources/application-test.yml

```diff
@@ -22,9 +22,8 @@ api.base.url:
   prison.person: http://localhost:8095
   casenotes: http://localhost:8096
   personal.relationships: http://localhost:8097
-  # for now use same Wiremock as personal.relationships - change later
-  organisations: http://localhost:8097
   nomis-sync: http://localhost:8098
+  organisations: http://localhost:8100
```

Comment:
8099 doesn't seem to work on my machine - skipped 1

Reply:
That's my port